### PR TITLE
Adding Bourbon, Neat, and Bitters.

### PR DESCRIPTION
**Pull request by lachlanhardy**
Making an Old Fashioned?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.sass-cache
```
Comment (elle) [inline .gitignore:1]: :+1:

File: base/_base.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Bitters 1.1.0
+// http://bitters.bourbon.io
+// Copyright 2013-2015 thoughtbot, inc.
+// MIT License
```
Comment (elle) [inline base/_base.scss:4]: Thoughts about removing these comments?

File: base/_base.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+@import "variables";
+
+// Neat Settings -- uncomment if using Neat -- must be imported before Neat
+// @import "grid-settings";
```
Comment (elle) [inline base/_base.scss]: There is a file for grid-settings but it is not imported. Should it be?

File: base/_base.scss (hunk 2 above)
Comment (lachlanhardy) [inline base/_base.scss:4]: I kinda figured that was up to thoughtbot :)

**Pull request merged**